style.css

```diff
@@ -1,7 +1,12 @@
 @import 'rest.css';
 @import 'map.css';
 @import 'galery.css';
+@import 'preloader.css';
 
 * {
   box-sizing: border-box;
 }
+
+.bodyApp {
```

Comment:
misleading class name ;) `body` suggests it's applied to `<body>` element. let's call it `main`, as we usually do